company/okrs/2020_q1.md

```diff
@@ -21,9 +21,7 @@
 1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
    1. VP Business (Success) KR: Generate at least [$M][M] in net new ARR from existing accounts.
    1. VP Business (Success) KR: Hire a Head of Customer Success who can own this function.
-   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers.
-
-1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers (by downloading and aggregating submissions from [HubSpot](https://app.hubspot.com/forms/2762526/a86bbac5-576d-4ca0-86c1-0c60837c3eab/submissions)). 
```

Comment:
I considered switching this to a usage growth metric, but it felt duplicative with the first bullet here. Do you think it should be removed?